api/src/main/resources/workbench-api.yaml

```diff
@@ -5362,15 +5362,17 @@ definitions:
         type: array
         items:
           "$ref": "#/definitions/Profile"
-  UpdateUserDisabledRequest:
+  UpdateDisabledStatusForUsersRequest:
     type: object
     required:
-    - email
+    - emails
     properties:
-      email:
-        type: string
+      emailList:
+        type: array
```

Comment:
please add a description indicating that these are usernames and not contact emails. Better yet, rename it.